_rules/inline_link_in_paragraph_is_distinguishable_be4d0c.md

```diff
@@ -0,0 +1,240 @@
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This is a link that is a descendant of a paragraph element and it uses the default styling of links which makes it underlined in most browsers which is a distinguish style.
+
+```html
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 2
+
+This is a link that is a descendant of a paragraph element and the underline is removed but an icon is added to denote it as a link.
+
+```html
+<html>
+	<head>
+		<title>Passed example 2 for rule be4d0c</title>
+		<style>
+			a.test {
+				text-decoration: none;
+			}
+		</style>
+	</head>
+	<body>
+		<p>
+			Read about WAI on the
+			<a class="test" href="http://w3.org/WAI">WAI webpage <img src="/test-assets/be4d0c/icon.png" alt=""/></a>.
+		</p>
+	</body>
+</html>
+```
+
+#### Passed Example 3
+
+This is a link that is a descendant of a paragraph element and the underline is removed but a text is added to denote it as a link.
+
+```html
+<html>
+	<head>
+		<title>Passed example 3 for rule be4d0c</title>
+		<style>
+			a.test {
+				text-decoration: none;
+			}
+		</style>
+	</head>
+	<body>
+		<p>Read about WAI on the <a class="test" href="http://w3.org/WAI">WAI webpage by following this link</a>.</p>
+	</body>
+</html>
+```
+
+#### Passed Example 4
+
+This is a link that is a descendant of a paragraph element and the underline is removed but the link has a distinguishing box-shadow which is a distinguishing style.
+
+```html
+<html>
+	<head>
+		<title>Passed example 4 for rule be4d0c</title>
+		<style>
+			a.test {
+				text-decoration: none;
+				box-shadow: 4px 4px;
+			}
+		</style>
+	</head>
+	<body>
+		<p>Read about WAI on the <a class="test" href="http://w3.org/WAI">WAI webpage</a>.</p>
+	</body>
+</html>
+```
+
+#### Passed Example 5
+
+This is a link that is a descendant of a paragraph element and the underline is removed but the link has a text contrast of more than 3:1 compared to the other text in the paragraph and when it receives focus or hover an underline appears.
+
+```html
+<html>
+	<head>
+		<title>Passed example 5 for rule be4d0c</title>
+		<style>
+			a {
+				text-decoration: none;
+				color: #d14826;
+			}
+			a:hover {
+				text-decoration: underline;
+			}
+			a:focus {
+				text-decoration: underline;
+			}
+		</style>
+	</head>
+	<body>
+		<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This is a link that is a descendant of a paragraph element and the underline is removed but the link has no other way of being recognized as a link.
+
+```html
+<html>
+	<head>
+		<title>Failed example 1 for rule be4d0c</title>
+		<style>
+			a.test {
+				text-decoration: none;
+			}
+		</style>
+	</head>
+	<body>
+		<p>Read about WAI on the <a class="test" href="http://w3.org/WAI">WAI webpage</a>.</p>
+	</body>
+</html>
+```
+
+### Inapplicable
```

Comment:
Inapplicable example 3 - 6 need to have underline removed for them to be a meaningful test.

Reply:
Do you mean to say that inapplicable examples should fail the SC if they would where to be applicable?

And if so, why only 3-6, why not also add this to 1 and 2?

Reply:
I do not see the need to remove the underline from examples 3 - 6. By keeping the underline, the test cases focus on the relevant parts of the (in)applicability.

I would suggest adding an inapplicable test case with a link that is not distunguised using colour. I see that this is handled as an assumption and may not need its own test case?